usaspending_api/search/v2/elasticsearch_helper.py

```diff
@@ -225,3 +227,288 @@ def concat_if_array(data):
             # This should never happen if TinyShield is functioning properly
             logger.error("Keyword submitted was not a string or array")
             return ""
+
+
+def get_base_search_with_filters(index_name: str, filters: dict) -> Search:
```

Comment:
The function this is based on isn't a good design. There is much room for improvement.